client/blocks/product-purchase-features/product-purchase-features-list/jetpack-wordpress-com.jsx

```diff
@@ -14,17 +14,17 @@ export default localize( ( { selectedSite, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="plugins"
-				title={ translate( 'Get the most from WordPress.com' ) }
+				title={ translate( 'Automatic Updates' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Enable Automatic Updates' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-myplan-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).